api/turing/service/ensembling_job_service.go

```diff
@@ -257,10 +251,6 @@ func (s *ensemblingJobService) generateMonitoringURL(job *models.EnsemblingJob,
 	}
 
 	name := job.Name
```

Comment:
Is this separate variable `name` still required?

Reply:
yes this is still needed to generate, but the caveat is that it should be called after db.Save() to trigger the name generation.

Reply:
MonitoringURL is still generated at runtime and not persisted into the database.

Reply:
Sorry, I meant do you still need `name := job.Name` here, or you can just use `job.Name` whenever you need that `name` variable (i.e. L256)

Reply:
Oh I see, just asthetic reasons. Yeah I just inlined it.